diagnostic.md

```diff
@@ -7,7 +7,18 @@ Assume all code executes in strict mode.
 
 ## Question 1
 
-Please list the JavaScript primitives and give one example of each.
+String: The String global object is a constructor for strings, or a sequence
+of characters.
+
+Number: The Number JavaScript object is a wrapper object allowing you to work
+with numerical values.
+
+Boolean: The Boolean object is an object wrapper for a boolean value true or
+false
+
+Null: Means null.
+
+Undefined: Means undefined.
```

Comment:
The question asked to give an EXAMPLE of each. 
String: "hi Brandon"
Number: 1
Boolean: true
Null: null
Undefined: undefined